modules/claims_api/spec/fixtures/form_526_json_api.json

```diff
@@ -16,6 +16,16 @@
         }
       },
       "claimantCertification": true,
+      "homelessness": {
+        "pointOfContact": {
+          "pointOfContactName": "John Doe",
+          "primaryPhone": "555-555-5555"
+        },
+        "currentlyHomeless": {
+          "homelessSituationType": "FLEEING_CURRENT_RESIDENCE",
+          "otherLivingSituation": "other living situations"
+        }
+      },
```

Comment:
same situation. needs to be moved under "veteran"

Reply:
Thanks for catching this.